### DISCUSSION TITLE
Code content definition: field indexing best practice

### DISCUSSION THREAD

**Discussion by PiemP** [OrchardCMS/OrchardCore#13281] [Q&A] [answered]
Hi all, 

I have see the warning about `ContentIndexSettings` to be obsolete in a future release. I would like to ask to you about the best practice to index fields or parts. In the first moment I believed that I can use directly `DocumentIndexOptions` but now I'm pretty sure that I have to use `LuceneContentIndexSettings` or `ElasticContentIndexSettings`. Can someone confirm this is correct?

Thank you.

**Comment by hishamco** [comment, marked as answer]
Use `LuceneContentIndexSettings` for Lucene Search and `ElasticContentIndexSettings` for Elastic Search
